pkg/apis/core/types.go

```diff
@@ -411,6 +411,9 @@ type PersistentVolumeClaimSpec struct {
 	// This is an alpha feature and may change in the future.
 	// +optional
 	VolumeMode *PersistentVolumeMode
+	// If specified, volume will be prepopulated with data from the DataSource.
```

Comment:
need to document something like "This depends on the provisioner for this volume being able to use the specified source.  If the provisioner does not support it, ...:

Reply:
Done